src/scheduler/resv_info.c

```diff
@@ -534,10 +534,10 @@ query_reservations(server_info *sinfo, struct batch_status *resvs)
 			}
 		}
 		resresv_arr[idx] = resresv;
+		resresv_arr[idx + 1] = NULL;
```

Comment:
I thought it'd be more clear doing it this way instead of writing:
resresv_arr[idx++] = resresv;
resresv_arr[idx] = NULL;

But I should follow scheduler semantics, so if you think they are both equally clear then I'll change it to the increment operator.